whois-commons/src/main/resources/nrtm_schema.sql

```diff
@@ -19,58 +35,72 @@ create table `source`
 ) ENGINE = InnoDB
   DEFAULT CHARSET = utf8;
 
-DROP TABLE IF EXISTS `version_information`;
-create table `version_information`
+CREATE TABLE `version_info`
 (
```

Comment:
Can we have drop table statement just before create table, makes it consistent with other schema and more readable?